CHANGELOG.md

```diff
@@ -1,3 +1,24 @@
+## 7.0.4 (YYYY-MM-DD)
+
+### Enhancements
+* None.
+
+### Fixes
+* In some cases a frozen Realm of the wrong version could be returned.
+* Upgrading files with string primary keys would result in a file where it was not possible to find the objects by primary key. ([Core issue #3893](https://github.com/realm/realm-core/pull/3893), since 7.0.0)
+
+### Compatibility
+* Realm Object Server: 3.23.1 or later.
+* File format: Generates Realms with format v10 (Reads and upgrades all previous formats from Realm Java 2.0 and later).
```

Comment:
How does this play with the below comment about "Fileformat has been bumped from 10 to 11."

Reply:
`2.0` is the release number which isn't associated with the file format number in any way.

Reply:
It wasn't the `2.0` but the `File format: Generates Realms with format v10` I was questioning.